lib/pbench/server/api/resources/upload_api.py

```diff
@@ -59,7 +59,7 @@ def put(self, controller):
             abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
 
         if not request.headers.get("filename"):
-            self.logger.debug(
+            self.logger.warning(
```

Comment:
This conflicts with Pete's changes. At one point when we'd talked about log level someone had proposed reserving `warning` for "user errors" we wanted to note but separate from internal Pbench errors; your usage of `warning` here fits better than Pete's `info`. I would propose that Pete just back out his log level changes given this PR.

Reply:
If fixed my changes to match.